en/manage-ha-postgresql.md

```diff
@@ -0,0 +1,126 @@
+Title: PostgreSQL HA notes | MAAS
+
+
+# PostgreSQL HA: hot standby
+
+**Disclaimer: These are *notes* that the reader can peruse to get an idea of
+what may be involved at the command line level when setting up 'hot standby' HA
+mode in PostgreSQL. The below procedures are not regularly tested. Please use
+the
+[PostgreSQL documentation](https://www.postgresql.org/docs/9.5/static/high-availability.html)
+when setting up HA on PostgreSQL.**
+
+<!-- NOTES
+
+https://www.postgresql.org/docs/9.5/static/high-availability.html
+https://jujucharms.com/postgresql/
+
+CREATETABLE guestbook (visitor_email text, vistor_id serial, date timestamp, message text);
+INSERT INTO guestbook (visitor_email, date, message) VALUES ( 'jim@gmail.com', current_date, 'This is a test.');
+INSERT INTO guestbook (visitor_email, date, message) VALUES ( 'jim@gmail.com', current_date, 'Now we are replicating.');
+SELECT * from guestbook;
+
+Primary host:
+This should change over time ('*_location' values):
+sudo -u postgres psql -x -c "select * from pg_stat_replication;"
+
+-->
+
+The following variables are used on this page:
+
+- PRIMARY_PG_IP: The IP address of the host that contains the primary database.
+- SECONDARY_PG_IP: The IP address of the host that contains the secondary database.
+- REP_USER: The internal database user that manages replication on the primary. 
+- REP_USER_PW: The password of the replication user.
+
+Their values are represented when they are preceded with the '$' character
+(e.g. $REP_USER_PW). These are to be replaced with actual values in the commands
+and files below.
+
+
+## Primary
+
+Perform these actions on the primary host.
+
+1) Create an internal database user to manage replication. You will be prompted
```

Comment:
Because these are terse notes it would be overkill. Applying a doc standard (which I didn't know about) would also make these notes look like actual documentation.